stdlib/LinearAlgebra/test/testgroups

```diff
@@ -24,3 +24,4 @@ pinv
 givens
 structuredbroadcast
 addmul
+ldlt
```

Comment:
I think `addmul` is supposed to be on the last line, so please insert `ldlt` somewhere in the middle.